Documentation/RunningTests.md

```diff
@@ -0,0 +1,76 @@
+## Running SerenityOS Tests
+
+There are two classes of tests built during a Serenity build: host tests and target tests. Host tests run on the build
+machine, and use [Lagom](https://github.com/SerenityOS/serenity/blob/master/Meta/Lagom/README.md) to build Serenity
```

Comment:
Let's link relative to this, so it works offline as well:

```suggestion
machine, and use [Lagom](../Meta/Lagom/README.md) to build Serenity
```

Reply:
sounds good. I think at least one of the other documentation pages has github links in it, so that's probably worth a docs fixup MR. Or at least, they did when I started this page :)